pyproject.toml

```diff
@@ -0,0 +1,271 @@
+[tool.poetry]
+name = "pec"
+version = "0.1.0"
+description = ""
+authors = "Krzysztof Wisniewski", "Mateusz Szpyruk", "Mateusz Zalewski", "Paweł Olszewski"
```

Comment:
Are U sure it is a valid TOML syntax? [See this.](https://python-poetry.org/docs/pyproject/#authors)